src/ray/raylet/node_manager.cc

```diff
@@ -2016,8 +2024,10 @@ void NodeManager::HandleTaskUnblocked(
   task_dependency_manager_.UnsubscribeGetDependencies(current_task_id);
```

Comment:
I think we need to do this for HandleDirectCallTaskUnblocked as well...

Reply:
Done